eng/helix/content/RunTests/RunTests.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
```

I shouldn't be surprised this didn't work because the project is intentionally isolated from our TFM hacks.
```suggestion
    <TargetFramework>net6.0</TargetFramework>
```